code/common/os.q

```diff
@@ -14,4 +14,5 @@ df:{(`$("/";"\\")[NT]sv -1_v;`$-1#v:("/";"\\")[NT]vs pth(string x;x)[10h=type x]
 run:{system"q ",x}
 kill:{[p]@[(`::p);"\\\\";1];}
 sleep:{x:string x; system("sleep ",x;"timeout /t ",x," >nul")[NT]}
-pthq:{[x] $[10h=type x;ssr [x;"\\";"/"];`$ -1 _ ssr [string (` sv x,`);"\\";"/"]]}
\ No newline at end of file
+pthq:{[x] $[10h=type x;ssr [x;"\\";"/"];`$ -1 _ ssr [string (` sv x,`);"\\";"/"]]}
+list:{$[NT;system"dir ",pth x;system"ls ",pth x]}
```

Comment:
is this meant to be in this PR? seems like an unrelated change, don't see it used elsewhere (plus it's easier to just use `key` anyway)

Reply:
it's strange to see this here. I thought I removed this in a previous pr. Will sort it out now.

Reply:
make sure that when you are creating a new branch, you do a `git checkout master` first, either `stash` or `checkout` any changes relative to master & then branch from there to ensure no unrelated changes end up in the new branch